game-core/src/main/java/games/strategy/triplea/ai/pro/data/ProPurchaseOption.java

```diff
@@ -252,6 +257,9 @@ public double getAmphibEfficiency(
   }
 
   private double calculateLandDistanceFactor(final int enemyDistance) {
+    if (movement <= 0) {
+      return 0.1;
```

Comment:
What's the rationale for 0.1?

Reply:
Want it to be essentially an order of magnitude worse than 1 move units. Don't want to return 0 as this is used to multiply with other values to get a kind of purchase value score for units.

Reply:
Would you mind adding a comment in code? That is actually a bit more involved than I had expected.

Reply:
Added.